tools/pylib/boututils/calculus.py

```diff
@@ -8,39 +8,30 @@
 from __future__ import division
 
 try:
-  from builtins import range
+    from builtins import range
 except:
-  pass
+    pass
 
 try:
-  from past.utils import old_div
+    from past.utils import old_div
 except:
-  def old_div(a,b):
-    return a / b
+    def old_div(a,b):
+        return a / b
```

Comment:
I think this is fine for now, but at some point removing old_div calls would be good.

Reply:
Oh, they really need to be removed.
They fall back to normal division - which is probably not what the intention was. Having always / would be better, then the errors would at least appear more often, and thus easier to notice + fix.

Should part of https://github.com/boutproject/BOUT-dev/commit/b3ec7246599af75a026c7bfad212d931fddf31a5 be undone?

Reply:
probably part of it... it depends if any code depends on the old behavior with integer division. In most cases it's safe to just replace `old_div(a, b)` with `(a/b)` but the occasional `int(a/b)` may be needed.